libmscore/ledgerline.cpp

```diff
@@ -67,9 +67,9 @@ void LedgerLine::layout()
             setColor(staff()->color());
       qreal w2 = _width * .5;
       if (vertical)
-            bbox().setRect(-w2, -w2, _width, _len + _width);
+            bbox().setRect(-w2, 0, w2, _len);
```

Comment:
The bounding box was too thick before already, by `_width/2`.
Now also the length got adjusted to account for the `Qt::FlatCap` not needed extra space